src/indigenousLayers.js

```diff
@@ -38,49 +37,35 @@ L.LayerGroup.IndigenousLandsTerritoriesLayer = L.LayerGroup.extend(
                 (function() {
                     var zoom = self._map.getZoom(), origin = self._map.getCenter() ;
                     var $ = window.jQuery;
+                    var ILL_url;
+
+                    if(this.name === "Territories"){
+                        ILL_url = "https://native-land.ca/api/index.php?maps=territories&position=" + parseInt(origin.lat) + "," + parseInt(origin.lng);
```

Comment:
How about using `eval` here instead of this if ladder?

Reply:
sure, but in that case won't all three conditions be run unnecessarily ? even if the first one is found to be true ?

Reply:
I've suggested some changes on the particular line of code in discussion above. Take a look!